Node/calling/tests/errors.js

```diff
@@ -1,121 +0,0 @@
-var assert = require('assert');
```

Comment:
Same as previous comments - this is covered in 'botbuilder' (core). UniversalBot not applicable for 'calling'.